pymc3/smc/sample_smc.py

```diff
@@ -157,6 +147,29 @@ def sample_smc(
         %282007%29133:7%28816%29>`__
     """
 
+    if kernel is not None:
```

Comment:
This should not be deprecated, we still want to have kernels.

Reply:
Yeah I know. We should just deprecate the "keywords" for the time being.